javaslang/src/test/java/javaslang/LazyTest.java

```diff
@@ -49,16 +49,16 @@ public void shouldMemoizeValues() {
 
     @Test
     public void shouldSequenceEmpty() {
-        final List<Lazy<Integer>> testee = List.empty();
+        final Seq<Lazy<Integer>> testee = Stream.empty();
```

Comment:
@ashrko619 looks like it has to be `Stream.empty()`. Do you have a pointer to the test (File/Line number)?